### PR TITLE
fix jasmine test heirarchy

### LINKED ISSUE #2493: Test names are inconsistent and entity controller tests do not fall under "Controller Tests"
* **Overview of the issue**

Jasmine tests can be organized in a hierarchical manner by nesting `describe`. This is not done properly in a generated app:

tests

You can see `Controller Tests` and `Controllers Tests `. None of the entity detail controller tests are in there, either.

* **Motivation for or Use Case** 

Makes test output harder to read.

* **JHipster Version(s)** 

2.25.0

* **Reproduce the error** 

Generate an app and run the tests. You will see `Controller Tests`, `Controllers Tests `, `Directive Tests`, and `Services Tests `. Any generated entities will also be unnested (see Address Detail Controller in the screenshot).

* **Suggest a Fix** 

By renaming all `Controllers Tests ` to `Controller Tests` and nesting entity tests in `describe('Controller Tests' ...`, the tests will be nested properly:

tests2

I can open a pull request.

### PR DESCRIPTION
**Pull request by geraldhumphries**
fix jasmine test heirarchy by fixing test names and moving entity
controller tests under `Controller Tests`

fix #2493

### CODE REVIEW

**Comment by geraldhumphries** [comment]
I have removed the "s" from tests that had it (Services Tests and Controllers Tests) and also removed the trailing spaces. The spaces may have been intentional - there may be a tool that relies on those spaces for properly formatted test output.

File: app/templates/src/test/javascript/spec/app/account/activate/_activate.controller.spec.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-describe('Controllers Tests ', function() {
+describe('Controller Tests', function() {
```
Comment (mraible) [inline app/templates/src/test/javascript/spec/app/account/activate/_activate.controller.spec.js]: I'd recommend going a step further and naming these something like 'Activate Controller Tests'.

Comment (geraldhumphries) [inline app/templates/src/test/javascript/spec/app/account/activate/_activate.controller.spec.js]: There is another describe in the file called "ActivationController". I can rename that.

**Comment by jdubois** [comment]
For the current naming: I don't remember any discussion about this, so I don't think there is any rational behind the current names.
So feel free to rename them, if that looks better!
-> is this ready to be merged?

**Comment by geraldhumphries** [comment]
Not yet, I would like to discuss the test names.

Most of the tests in the main generator are named after the directive/controller/service they are testing. ActivationController, HealthController, etc. Some are named after their function ("Password strength"). The entity detail controller tests are named "Entity Detail Controller". I think we should standardize to naming it after the directive/controller/service itself, so "Entity Detail Controller" would become "EntityDetailController" and "Password strength" would become "passwordStrengthBar".

**Pull request merged**